lib/generators/graphiti/resource_generator.rb

```diff
@@ -21,9 +21,10 @@ class ResourceGenerator < ::Rails::Generators::NamedBase
       aliases: ['--actions', '-a'],
       desc: 'Array of controller actions to support, e.g. "index show destroy"'
 
-    class_option :'default-attributes-class',
+    class_option :'attributes-from',
       banner: 'Model',
       type: :string,
+      alias: ["--attributes-from", ""]
```

Comment:
How about `--model` and `-m`, to note we'll grab attributes by introspecting that model?